exporter/chronicleexporter/exporter.go

```diff
@@ -113,6 +124,7 @@ func (ce *chronicleExporter) logsDataPusher(ctx context.Context, ld plog.Logs) e
 			continue
 		}
 
+		ce.logger.Debug("Sending payload to Chronicle", zap.ByteString("payload", data))
```

Comment:
Should we be logging this? Could there be sensitive data in the payload?

Reply:
yeah now that I think about it it does have the customer id which you can't really do anything with on its own but probably still not great. removed.